foreman/data_refinery_foreman/foreman/test_end_to_end.py

```diff
@@ -302,3 +305,58 @@ def test_all_the_things(self):
         self.assertEqual(
             Sample.processed_objects.exclude(accession_code__in=SMASHER_SAMPLES).count(), 154
         )
+
+        # Make sure that a processed file using our new transcriptome index is
+        # similar enough to a reference file
+        print("Now we are going to verify that the outputs look okay")
+
+        sample = Sample.objects.get(accession_code="SRR5085168")
+
+        # First, sanity check that the genome build hasn't changed. If it
+        # changes, all bets are off when comparing quant.sf files.
+        self.assertEqual(
+            sample.results.all()
+            .filter(processor__name=ProcessorEnum.SALMON_QUANT.value["name"])
+            .first()
+            .organism_index.assembly_name,
+            "R64-1-1",
+        )
+        # Note that the reference file was processed using version 96
+        self.assertEqual(
+            sample.results.all()
+            .filter(processor__name=ProcessorEnum.SALMON_QUANT.value["name"])
+            .first()
+            .organism_index.release_version,
+            "104",
```

Comment:
This number changes over time as new assemblies are released. We get this number by querying ensembl's API because we used to have it hardcoded and it would break every now and then when a new version was released. It seems this test will now do the same.

Reply:
My thinking here was that this falls under the category of "things we want to know about", but I can remove this check so we are just notified when the genome build itself changes.

@jaclyn-taroni which would you prefer?

Reply:
Genome only build sounds good to me 👍🏻  ideally we'd process everything with the same release, but I don't know that that is realistic.

Reply:
I don't think it is. The API for the current and past releases are not consistent. We potentially could change things to use the API for a pinned past release, but that would mean we couldn't use the latest if we wanted to...